source/server/listener_manager_impl.cc

```diff
@@ -709,7 +710,7 @@ void ListenerManagerImpl::startWorkers(GuardDog& guard_dog) {
     }
     worker->start(guard_dog);
     if (enable_dispatcher_stats_) {
-      worker->initializeStats(*scope_, fmt::format("worker_{}.", i++));
+      worker->initializeStats(*scope_, fmt::format("worker_{}.", i));
```

Comment:
This is an actual change, right? It changes the stats from 0 index to 1 index?

Reply:
Ah..I see. I did not realize that. Sorry.

Reply:
Is there some test that would have caught this somehow? That would be optimal? Maybe in a test that enables dispatcher stats in server_test?

/wait

Reply:
There is no test in server_test that enables dispatcher stats. Also since all of the stats are histograms we can not verify them in server_test because it uses IsolateStore which does not support histograms. 

I have changed the `listener_manager_impl_test` to verify the prefix. PTAL.